index.html

```diff
@@ -68,6 +67,123 @@ <h2 id="support-text">I'm a Software Developer</h2>
     <img id="main-img" src="assets/illustration.svg" alt="">
   </section>
 
+  <section id="recent-works">
+    <div class="container">
+      <div class="section-title">
+        <h1 id="recent-works-title">My Recent Works</h1>
+      </div>
+      <div class="line"></div>
+      <img id="recent-works-img" src="assets/img-placeholder.png" alt="">
+      <div id="post-stories">
+        <h1>Multi-Post Stories</h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+          standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <div class="row-container">
+          <div class="text-box">
+            <p>CSS</p>
+          </div>
+          <div class="text-box">
+            <p>HTML</p>
+          </div>
+          <div class="text-box">
+            <p>Bootstrap</p>
+          </div>
+          <div class="text-box">
+            <p>Ruby</p>
+          </div>
+        </div>
+        <div class="orange-button">
+          <a class="btn-text">See project</a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section id="work1">
+    <div class="container">
+      <div id="post-stories">
+        <h1>Profesional Art Printing Data</h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's
+          standard.</p>
+        <div class="row-container">
+          <div class="text-box">
+            <p>HTML</p>
+          </div>
+          <div class="text-box">
+            <p>Bootstrap</p>
+          </div>
+          <div class="text-box">
+            <p>Ruby</p>
+          </div>
+        </div>
+        <div class="fw-orange-button">
+          <a class="btn-text">See project</a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section id="about-me">
+    <div class="container">
+      <div class="section-title">
+        <h1 id="about-me-title">About me</h1>
+      </div>
+      <div id="about-me-text">
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my
+          work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <div class="orange-button">
+          <a class="btn-text">Get my resume</a>
+        </div>
+        <img id="about-me-img" src="assets/Frame.svg" alt="">
+      </div>
+
+
+    </div>
+    <div class="line"></div>
+  </section>
+
+  <section id="skills">
+    <div class="grid-container">
+      <div class="skills-box">
```

Comment:
- [ ] I kindly suggest that you also use a two level nested `ul` `li` because the technology categories ("Languages", "Frameworks" and "Skills") should also be in a `ul`, each of them in a `li`. Kindly see the example below.

## EXAMPLE
```
   <ul>
        <li class="languague">
              <ul>
                  <li><a href="#">JavaScript</a></li>
                  <li><a href="#">HTML</a>
                  <li><a href="#">CSS</a>
             </ul>
         </li>
          <li class="framework">
              <ul>
                  <li><a href="#">Bootstrap</a></li>
                  <li><a href="#">Ruby on Rails</a>
                  <li><a href="#">RSPEC</a>
             </ul>
         </li>
            <li class="skills">
              <ul>
                  <li><a href="#">Code Kit</a></li>
                  <li><a href="#">GitHub</a>
                  <li><a href="#">Codepen</a>
             </ul>
         </li>                                     
</ul>
```

Please note that the above is just an example and may not apply directly to your code.